src/api/unified/lapack.cpp

```diff
@@ -83,7 +83,7 @@ af_err af_pinverse(af_array *out, const af_array in, const double tol,
                    const af_mat_prop options)
 {
     CHECK_ARRAYS(in);
-    return CALL(out, in, options);
+    return CALL(out, in, tol, options);
```

Comment:
that an actual bug.

Reply:
Yeah, we need stronger types internally. Should I submit this separately?

Reply:
A separate commit should be fine, I am fine with it in this PR.